projects/vs/psa-boot-test/psa-boot-test.cpp

```diff
@@ -5,72 +5,121 @@
 
 using namespace Microsoft::VisualStudio::CppUnitTestFramework;
 
-namespace psaboottest
+#define GET_TEST_PACKAGE(_pack)                (boot_upgrade_package_t*)copy_package(_pack, sizeof(_pack))
+#define RELEASE_TEST_PACKAGE(_pack)            free(_pack)
+
+/// <summary>
+/// Copies a package into Read/Write Buffer. 
+/// Example Tests packages are constant values so
+/// decryption cannot update the packages, therefore,
+/// we need to copy the package to a updatable memory.
+/// </summary>
+/// <param name="source">Source Package</param>
+/// <param name="size">Source Package Size</param>
+/// <returns>Destination Package</returns>
+uint8_t* copy_package(const uint8_t* source, uint32_t size)
+{
+    uint8_t* dest = (uint8_t*)malloc(size);
+    if (dest == NULL)
+    {
+        return NULL;
+    }
+
+    memcpy(dest, source, size);
+
+    return dest;
+}
+
+/// <summary>
+/// Compares two strings and returns true if strings are equals
+/// </summary>
+/// <param name="str1">String 1</param>
+/// <param name="str2">String 2</param>
+/// <returns>True if strings are equal, otherwise returns false</returns>
+bool arrays_are_equal(const uint8_t* arr1, const uint8_t* arr2, uint32_t len)
```

Comment:
Naming suggestion is_arrays_are_equal. 
For me arrays_are_equal is not comparison it is directly equalising two arrays.

Reply:
Make sense. Let fix it. 
`are_arrays_equal()`